types/react/index.d.ts

```diff
@@ -310,7 +310,7 @@ declare namespace React {
 
     type SFC<P = {}> = StatelessComponent<P>;
     interface StatelessComponent<P = {}> {
```

Comment:
This shouldn't be here. What happened to `FunctionComponent`?

It looks like this is an incredibly old commit.